templates/tabby-web.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>tabby-web</Name>
+  <Repository>ghcr.io/eugeny/tabby-web:latest</Repository>
+  <Registry>https://hub.docker.com/r/outlinewiki/outline/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/Eugeny/tabby-web/issues/new</Support>
+  <Project>https://tabby.sh</Project>
+  <Overview>Tabby Web - an SSH/Telnet/Serial client in your browser.&#xD;</Overview>
+  <Category>Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:9090]</WebUI>
+  <Requires>MariaDB&#xD;</Requires>
+  <Config Name="Port HTTP" Target="80" Default="9090" Mode="tcp" Description="" Type="Port" Display="always" Required="true" Mask="false">9090</Config>
+  <Config Name="Database URL" Target="DATABASE_URL" Default="mysql://root:123@db/tabby" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">mysql://root:123@db/tabby</Config>
+  <Config Name="Debug" Target="DEBUG" Default="false" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">false</Config>
```

Comment:
I'm not sure if this value needs to specifically be `False` with a capital "F": https://github.com/Eugeny/tabby-web/blob/abcdfbcd4e003168189d6698db901e50ca39a1c0/docker-compose.yml#L12

Reply:
it doesn't seem to require a capital "F". I tried both ways and didn't see a difference in the container itself.